enterprise/api/v1/views.py

```diff
@@ -525,6 +526,39 @@ def unlink_users(self, request, pk=None):  # pylint: disable=unused-argument
 
         return Response(status=HTTP_200_OK)
 
+    @action(detail=False)
+    def algolia_key(self, request, *args, **kwargs):
```

Comment:
Nit: the [doc](https://www.algolia.com/doc/api-reference/api-methods/generate-secured-api-key/#about-this-method) indicates that the key generation doesn't make any calls to the Algolia servers, so it is not that time-consuming, but I am wondering if we should add a per-user cache to this view to avoid the DB call.

Reply:
@CefBoud, I'm afraid a set of linked enterprise customers uuids _are_ cache key here, so I think we can't avoid DB call here.

Reply:
>But I guess we'll face the same problem of staleness as the one in the frontend.

Right, what I meant is that we must invalidate the cache if the set of linked enterprises changes.